swagger.yml

```diff
@@ -571,6 +571,107 @@ paths:
           description: "Not found"
           schema:
             $ref: "#/definitions/NotFound"
+  /groups/{groupID}/containers:
+    get:
+      operationId: getContainers
+      description: Get All Containers
+      parameters:
+        - name: groupID
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: "Success"
+          schema:
+            $ref: "#/definitions/GetContainersResponse"
+        401:
+          description: "Unauthorized"
+          schema:
+            $ref: "#/definitions/Unauthorized"
+        404:
+          description: "Not found"
+          schema:
+            $ref: "#/definitions/NotFound"
+    post:
```

Comment:
Could this post follow the idempotent `PUT` model using `/groups/{groupID}/containers/{containerID}`